app/components/mvc.js

```diff
@@ -1 +0,0 @@
-../../node_modules/fxos-mvc/dist/mvc.js
```

Comment:
That the thing I wanted to do for a long time, now we copy these external components at the build stage.